docs/app/components/navbar/navbar-demo.component.html

```diff
@@ -40,6 +45,17 @@ <h5>Example</h5>
         </hc-popover-content>
     </hc-tile>
 
+    <hc-tile>
+        <h5>Responsive Behavior</h5>
```

Comment:
I am a little confused on what belongs in the overview tab and what belongs in the documentation. Just out of curiosity, why would this be here instead of documentation?

Reply:
Thanks for the review @benjanderson and sorry it's taken me so long to get back to it.  As @jtrujill has already brought up in a recent issue, this question comes back to a fundamental question about who the demo site is for.  The reason I included it over here was because I was thinking product managers who might not want to click into the documentation, but would still have an interest in understand its responsive behavior.  So I was thinking of it more as an overview.  I think coming to a consensus on our target audience will be one of the first jobs of the steering committee, and that should give us some guidance around what content to include and where.  So I'll probably leave this tile as is until we reach that point, and then we can revisit this and others.